### PR TITLE
DOC: update fixing unknown parameters errors (error code PR02)

### PR DESCRIPTION
**Pull request by tolaa001**
- Documentation update fixing some of the methods with a PR-2 error code which involved updating **kwds to **kwargs

- No tests required

- The data frame.plot methods now have the docstring arguments updated from **kwds to **kwargs to match the method signature and exclude them from the PR02 errors:

### CODE REVIEW

File: pandas/plotting/_core.py [hunk 1]
```diff
@@ -1148,7 +1148,7 @@ def kde(self, bw_method=None, ind=None, **kwargs):
             1000 equally spaced points are used. If `ind` is a NumPy array, the
             KDE is evaluated at the points passed. If `ind` is an integer,
             `ind` number of equally spaced points are used.
-        **kwds : optional
+        **kwargs : optional
```
Comment (jbrockmendel) [inline pandas/plotting/_core.py]: should the "optional" part be consistent across all of these?

**Review by datapythonista — APPROVED** [review]
thanks for fixing those, lgtm

but if you don't mind removing the optional in this same PR, that would be great (also ok in a follow up)

**Comment by tolaa001** [comment]
Thanks for approving this Marc 😊. I will do this once am reunited with my laptop tomorrow evening.

Also thanks for all your advise at the Iwoca meet up last week.

**Comment by WillAyd** [comment]
Any reason why we don't change all instances of `kwds` to `kwargs` in this PR?

**Comment by datapythonista** [comment]
I'm not sure what's wrong with the CI. You'll have to debug `scripts/validate_docstring.py`. We don't want `**kwargs` to have a type in the docstring, if the script is failing because of it, it's a bug that needs to be fixed. Feels a bit weird that seems to be failing just for some of the cases.

**Comment by tolaa001** [comment]
Happy to take a look at `scripts/validate_docstring.py`.

**Comment by tolaa001** [comment]
@datapythonista can you please providing me with some guidance as I am unsure why the CI build is failing?
I have fixed the reason it was failing yesterday, it was because I had "`" before `kwargs`

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jorisvandenbossche** [comment]
@tolaa001 Thanks a lot for the contribution!

**Pull request merged**